fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStoreIndexTest.java

```diff
@@ -2707,182 +2747,6 @@ record = TestRecords1Proto.MySimpleRecord.newBuilder()
         }
     }
 
-    @Test
-    public void testBoundaryPrimaryKeys() {
```

Comment:
It seems like we should keep these tests until the methods are actually removed

Reply:
✅